src/main/java/me/zeroeightsix/kami/module/modules/player/ViewLock.kt

```diff
@@ -15,25 +17,24 @@ import kotlin.math.*
 )
 object ViewLock : Module() {
 
-    private val deltaXQueue: Queue<Pair<Int, Long>> = LinkedList<Pair<Int, Long>>(listOf())
-    private val deltaYQueue: Queue<Pair<Int, Long>> = LinkedList<Pair<Int, Long>>(listOf())
-    private var pitchSliceAngle = 1.0f
-    private var yawSliceAngle = 1.0f
     val yaw = register(Settings.b("Yaw", true))
     val pitch = register(Settings.b("Pitch", true))
     val autoYaw = register(Settings.booleanBuilder("AutoYaw").withValue(true).withVisibility { yaw.value })
     val autoPitch = register(Settings.booleanBuilder("AutoPitch").withValue(true).withVisibility { pitch.value })
     private val threshold = register(Settings.integerBuilder("ChangeThresh").withValue(5).withRange(1, 50).withStep(1).withVisibility { autoPitch.value || autoYaw.value })
     val disableMouseYaw = register(Settings.booleanBuilder("DisableMouseYaw").withValue(true).withVisibility { yaw.value })
     val disableMousePitch = register(Settings.booleanBuilder("DisableMousePitch").withValue(true).withVisibility { pitch.value })
-    private val specifcYaw = register(Settings.floatBuilder("SpecificYaw").withValue(180.0f).withRange(-180.0f, 180.0f).withStep(1.0f).withVisibility { !yaw.value })
-    private val specificPitch = register(Settings.floatBuilder("SpecificPitch").withValue(0.0f).withRange(-90.0f, 90.0f).withStep(1.0f).withVisibility { !pitch.value })
+    private val specifcYaw = register(Settings.floatBuilder("SpecificYaw").withValue(180.0f).withRange(-180.0f, 180.0f).withStep(1.0f).withVisibility { !autoYaw.value })
+    private val specificPitch = register(Settings.floatBuilder("SpecificPitch").withValue(0.0f).withRange(-90.0f, 90.0f).withStep(1.0f).withVisibility { !autoPitch.value })
     private val yawSlice = register(Settings.integerBuilder("YawSlice").withValue(8).withRange(2, 32).withStep(1).withVisibility { autoYaw.value })
     private val pitchSlice = register(Settings.integerBuilder("PitchSlice").withValue(5).withRange(2, 32).withStep(1).withVisibility { autoPitch.value })
 
     private var yawSanp = 0
     private var pitchSnap = 0
-
+    private val deltaXQueue: Queue<Pair<Int, Long>> = LinkedList()
+    private val deltaYQueue: Queue<Pair<Int, Long>> = LinkedList()
```

Comment:
Move the type parameter to the LinkedList constructor function 
```suggestion
    private val deltaXQueue = LinkedList<Pair<Int, Long>>()
    private val deltaYQueue = LinkedList<Pair<Int, Long>>()
```